site.region

```diff
@@ -347,7 +347,7 @@
             </p>
           </div>
           <div class="site-footer__badgerIcon" title="Sally the Red Badger Badger">
-            <img src="/assets/badger-on-black.svg">
+            <img src="/assets/badger-on-black.svg" alt="Sally the badger image">
```

Comment:
Maybe consider removing "image" from the alt text, as the screen reader will say that it is an image